packages/web/src/components/premium-content-purchase-modal/pages/GuestCheckoutPage.tsx

```diff
@@ -90,7 +100,21 @@ export const GuestCheckoutPage = (props: GuestCheckoutProps) => {
           </Text>
 
           <EmailField name={GUEST_EMAIL} helperText={''} />
-          {error ? <Hint>{error}</Hint> : null}
+          {error ? (
```

Comment:
Any chance we can reuse the "new email" component?

Reply:
good idea made some small adjustments to make this work